src/m_files.cc

```diff
@@ -1025,13 +1035,30 @@ void Instance::M_WriteEurekaLump(Wad_file *wad) const
 	if (!loaded.portName.empty())
 		lump->Printf("port %s\n", loaded.portName.c_str());
 
+        //remember pwd
+        static char old_dir[FL_PATH_MAX];
+
+        if (getcwd(old_dir, sizeof(old_dir)) == NULL)
+        {
+                old_dir[0] = 0;
+        }
+
+        SString current_pwad_path = FilenameGetPath(wad->PathName());
+        FileChangeDir(current_pwad_path);
```

Comment:
I'd rather that instead of temporarily changing the current directory so we can get the relative path from it, that instead we just apply the relative path on top of the PWAD path. I hope that's not made too complicated by the OS differences…